app/Rest/Employees_Controller.php

```diff
@@ -0,0 +1,475 @@
+			]
+		);
+	}
+
+	/**
+	 * Checks if a given request has access to create a post.
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return true|\WP_Error True if the request has access to create items, WP_Error object otherwise.
+	 */
+	public function create_item_permissions_check( $request ) {
+		if ( ! empty( $request['id'] ) ) {
+			return new \WP_Error( 'rest_post_exists', __( 'Cannot create existing post.' ), array( 'status' => 400 ) );
+		}
+
+		return true;
+	}
+
+	/**
+	 * Get_item
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return \WP_Error|\WP_REST_Response
+	 */
+	public function get_item( $request ) {
+		$post = $this->get_employee( $request['id'] );
+		if ( is_wp_error( $post ) ) {
+			return $post;
+		}
+
+		return $this->response( $post );
+	}
+
+	/**
+	 * Get_item
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return \WP_Error|\WP_REST_Response
+	 */
+	public function update_item( $request ) {
+		$prepared_post = $this->prepare_item_for_database( $request );
+
+		if ( is_wp_error( $prepared_post ) ) {
+			return $prepared_post;
+		}
+
+		$prepared_post->post_type   = Employee_Post_Type::$ID;
+		$prepared_post->post_status = Employee_Post_Type::STATUS_PUBLISH;
+
+		$post_id = wp_update_post( wp_slash( (array) $prepared_post ), true );
+
+		if ( is_wp_error( $post_id ) ) {
+			$post_id->add_data( array( 'status' => 400 ) );
+
+			if ( 'db_insert_error' === $post_id->get_error_code() ) {
+				$post_id->add_data( array( 'status' => 500 ) );
+			}
+
+			return $post_id;
+		}
+
+		$schema = $this->get_item_schema();
+
+		if ( ! empty( $schema['properties']['acf_field'] ) && isset( $request['acf_field'] ) ) {
+			$acf_update = $this->update_acf_value( $request['acf_field'], $post_id );
+
+			if ( is_wp_error( $acf_update ) ) {
+				return $acf_update;
+			}
+		}
+
+		$request->set_param( 'context', 'edit' );
+
+		return $this->response(
+			[
+				'status'      => 'Employee has been updated',
+				'id'          => $post_id,
+				'title'       => ! empty( $prepared_post->post_title ) ? $prepared_post->post_title : '',
+				'post_status' => $prepared_post->post_status,
+			]
+		);
+	}
+
+	/**
+	 * Get_item
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return \WP_REST_Response
+	 */
+	public function delete_item( $request ) {
+		$request->set_param( 'context', 'edit' );
+
+		$result = wp_delete_post( $request['id'], true );
+
+		if ( is_wp_error( $result ) ) {
+			return $result;
+		}
+
+		return $this->response(
+			[
+				'status' => 'Employee has been deleted',
+				'id'     => $request['id'],
+			]
+		);
+	}
+
+	/**
+	 * Item_permissions_check
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return bool|\WP_Error True if the request has read access for the item, WP_Error object otherwise.
```

Comment:
please set correct comments. As I understood - you just check this item exists in database, nothing more.

so please write good description instead of "Item_permissions_check".

and we don't need description for return parameter I think